internal/server/manager.go

```diff
@@ -79,17 +96,38 @@ func GetManagerInstance() *instance {
 }
 
 func (m *instance) GetManagedSveltosClusters() map[corev1.ObjectReference]ClusterInfo {
-	lock.Lock()
-	defer lock.Unlock()
+	lock.RLock()
```

Comment:
This is my bad (sorry bug was in my code). This should be clusterMux

The lock mutex is only to get the manager instance.